scripts/npm/prepublish/lib/changelog.js

```diff
@@ -1,21 +1,26 @@
-'use script';
-
 const fs = require( 'fs' );
-const path = require( 'path' );
 const promisify = require( 'promisify-node' );
 
 promisify( fs.writeFile );
 
-function updateChangelog( version ) {
-  const file = path.join( __dirname, '..', '..', '..', '..', 'CHANGELOG.md' );
-  const date = new Date().toJSON().slice( 0, 10 );
-  const latest = '## Unreleased\n\n### Added\n- \n\n### Changed\n- ' +
-               '\n\n### Removed\n- \n\n## ' + version + ' - ' + date;
+/**
+ * Updates the Unreleased seciton of the changelog to a versioned Release
```

Comment:
`section`

Reply:
Also, unreleased and released can be lowercase I think. Also, add a trailing period.

Reply:
Since we're referring to headings in the changelog I think capitalized makes sense